src/reports/IdReportGenerator.js

```diff
@@ -0,0 +1,24 @@
+import moment from 'moment';
+import { exportCsv } from '@folio/stripes/util';
+
+class IdReportGenerator {
+  constructor(fileNamePrefix) {
+    this.fileNamePrefix = fileNamePrefix;
+  }
+
+  parse(records, recordFinder = record => record) {
+    return records.map(record => ({ id: recordFinder(record) }));
+  }
+
+  toCSV(records, recordFinder) {
```

Comment:
We don't need `recordFinder` parameter since we're not using it here
Also, I think it'll be better to make this a hook `useReportGenerator` instead of a js class

Reply:
That's a good point. I didn't thought about `useReportGenerator` hook